cerbos/grpc.go

```diff
@@ -139,6 +141,10 @@ func New(address string, opts ...Opt) (*GRPCClient, error) {
 		return nil, err
 	}
 
+	if err := validator.Init(); err != nil {
```

Comment:
Looking at the validator code, the chances of this returning an error is quite slim. So, it's probably OK to use the lazy loading method I suggested below. No need for a global `Init` function.